rdr/busybox/scripts/kernel_untar.sh

```diff
@@ -69,6 +69,21 @@ set_running_state()
     sync -f $MOUNT
 }
 
+# Run kernal untar
+untar_kernal_file()
+{
+    # Sleeping for some random interval of time to reduce stress
+    echo "Sleeping for $sleep_time"
+    sleep $sleep_time
```

Comment:
@prsurve , do we want to use same time every iteration? OR do we want to change sleep time every iteration?